src/custom-elements/src/components/expansion-panel/expansion-panel.tsx

```diff
@@ -0,0 +1,89 @@
+import { Component, Event, EventEmitter, Prop } from '@stencil/core';
+
+@Component({
+    tag: 'chi-expansion-panel',
+    styleUrl: 'expansion-panel.scss',
+    scoped: true
+})
+export class ExpansionPanel {
+    /**
+     * to set expansion panel to 'done' state
+     */
+    @Prop({ reflectToAttr: true }) done?: boolean;
+
+    /**
+     * to set expansion panel to 'active' state
+     */
+    @Prop({ reflectToAttr: true }) active?: boolean;
+
+    /**
+     * to set expansion panel to 'disabled' state
+     */
+    @Prop({ reflectToAttr: true }) disabled?: boolean;
+
+    /**
+     * to set a step number next to the title of the panel
+     */
+    @Prop({ reflectToAttr: true }) step?: string;
+
+    /**
+     * to set the title of the panel
+     */
+    @Prop({ reflectToAttr: true }) eptitle: string;
+
+    /**
+     * is the panel border-styled?
+     */
+    @Prop({ reflectToAttr: true }) bordered?: boolean;
+
+    /**
+     * event raised when 'change' button of done mode panel is clicked
+     */
+    @Event({ eventName: 'chiEpChangeClick' }) chiEpChangeClick: EventEmitter;
```

Comment:
@srinath-koneru , can you validate this event name, please?

Reply:
@SergioQCL @dani-cl-madrid as we are triggering an event onClick I would say use chiClick as it is a button. You can just change the eventName and use the variable chiEpChangeClick in your code. If chiClick doesn't suit then we shall look for something else but it should not have CE acronym and definitely not two event names.

Reply:
@srinath-koneru @SergioQCL It is not a click in the component. It is a click in a very specific part of the component, so I think `chiClick` doesn't fit in here. How about `chiSetActiveState`?

Reply:
@srinath-koneru @SergioQCL Even better, `chiStateChanged` and set 'active' as the detail content of the event.

Reply:
@srinath-koneru, we've been talking about not forcing to include a button here and leaving a slot available for all states, so we provide more flexibility in our component as well as leave all the event logic to developers who want to include it. Thoughts?